addons/payment/views/payment_provider_views.xml

```diff
@@ -95,10 +95,24 @@
                         <page string="Fees" name="fees" attrs="{'invisible': [('support_fees', '=', False)]}">
                             <group name="payment_fees">
                                 <field name="fees_active"/>
-                                <field name="fees_dom_fixed" attrs="{'invisible': [('fees_active', '=', False)]}"/>
-                                <field name="fees_dom_var" attrs="{'invisible': [('fees_active', '=', False)]}"/>
-                                <field name="fees_int_fixed" attrs="{'invisible': [('fees_active', '=', False)]}"/>
-                                <field name="fees_int_var" attrs="{'invisible': [('fees_active', '=', False)]}"/>
+                                <field name="fees_dom_fixed"
+                                       widget="monetary"
+                                       style="max-width: 5rem"
```

Comment:
This has no effect on monetary fields.

Reply:
I'm pretty sure it used to. I'll check that with JS team

Reply:
Maybe I forgot to remove my local tweaks before assessing that... Let's check!

Reply:
No need to ask about this one: it does have an effect of reducing the length of the field.
![image](https://user-images.githubusercontent.com/112173839/204845329-76543236-4220-40b4-8c26-29e0ff568b5b.png)